### ISSUE TITLE
Quadrat: Showing "Comments are closed" in pages where it isn't supposed to

### ISSUE THREAD

**Issue by beafialho** [Automattic/themes#4750]
Right now, if comments are turned off and there are no comments it displays “comments are closed” anyway. Can we remove it?

<img width="1270" alt="Captura de ecrã 2021-09-30, às 13 02 54" src="https://user-images.githubusercontent.com/65220155/135451574-0cd53b32-5330-4ea7-984d-2811bbe7a753.png">

<img width="870" alt="Captura de ecrã 2021-09-30, às 13 02 35" src="https://user-images.githubusercontent.com/65220155/135451584-7188f48e-f131-4305-845e-c5652e507a3a.png">

Even though this may be a Gutenberg bug, if we can’t get it fixed there before Quadrat’s launch, how do you recommend hiding this in the meantime?

**Comment by mikachan** [comment]
I think we've previously fixed this in other themes by removing the comments block from the page template altogether, but then I can see we added them back to fix another issue which was requesting comments to show on a page: https://github.com/Automattic/themes/pull/4632

I think the best thing is to fix this in Gutenberg. The only way I can think for hiding them now is to remove comments from all pages (like we did before). I guess it depends on whether we want to see the 'Comments are closed' message or not see any comments at all, even if they're turned on?

If comments are turned off, it looks like they're inside a `.nocomments` class which we could hide with CSS, but I guess this is hacky.
